thorlcr/master/thmastermain.cpp

```diff
@@ -780,6 +782,22 @@ int main( int argc, char *argv[]  )
 
         if (registry->connect())
         {
+            if (channelsPerSlave > 1)
+            {
+                for (unsigned s=0; s<slavesPerNode; s++)
```

Comment:
You probably meant to list for all slave processes?
i.e. In a e.g. 50 way cluster with slavesPerNode=1 - this is only going log "Slave log 1" and not anything for slave's 2-50.

You should use queryNodeClusterWidth() instead of slavesPerNode.